pino.d.ts

```diff
@@ -13,193 +13,204 @@
 //                 Austin Beer <https://github.com/austin-beer>
 //                 Michel Nemnom <https://github.com/Pegase745>
 //                 Igor Savin <https://github.com/kibertoad>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 3.0
+//                 James Bromwell <https://github.com/thw0rted>
+// TypeScript Version: 4.4
 
-/// <reference types="node"/>
+import type { EventEmitter } from "events";
+import type { PrettyOptions } from "pino-pretty";
```

Comment:
I think we mostly leaned in the direction of inlining types for dependencies we don't already rely on in the past, so I would suggest doing the same here. However suboptimal it is, other solutions seem worse.

Reply:
Based on [this guidance](https://stackoverflow.com/a/63905376/26286), it sounds like the type import should gracefully degrade to `any` if pino-pretty isn't installed, which I think is the desired behavior -- if they aren't using the dependency, they shouldn't provide an argument for `prettyOptions`.  Is that what you meant by "seem worse" or is there some other (mis-)behavior I'm not aware of?

Reply:
I did test it and the behavior I described is accurate.  I ran `npm pack` on my fork, then `npm install typescript /path/to/pino-x.y.z.tgz` in a new project.  `pino-pretty` is not installed, and the language service shows the type of `prettyOptions` as `any`.  If you run `tsc` without `skipLibCheck`, it flags the `import type` line above, but also 6 other imports (for `node` / `NodeJS`,  `http`, `events`, and `worker_threads`).  I get that `pino` is mostly a Node-focused library but it does have a `browser` option, and requiring that users include `@types/node` to use it can cause conflicts (e.g., W3C and Node have different global `ReadableStream` interfaces).  IME, `skipLibCheck` is the least-worst solution.

Reply:
I've cracked it!  Adding a `// @ts-ignore` immediately before the `import type` line causes it to gracefully degrade to `any` when `pino-pretty` is not installed, even `skipLibCheck=false`.  Installing `pino-pretty` causes incorrect options (like `colorize: "true"`) to be flagged as an error, then you can uninstall `pino-pretty` and the code compiles cleanly.  (That's correct, right? Without `pino-pretty` installed, `prettyOptions` should just be ignored?)